packages/quantic/force-app/main/default/lwc/quanticSort/e2e/sortObject.ts

```diff
@@ -0,0 +1,76 @@
+import {Locator, Page, Request} from '@playwright/test';
```

Comment:
we agreed here to name all pageObjects `pageObject.ts` instead of the `componentNameObject.ts` (not your fault we agreed after your PR was done)

I would change the name of the file, you can keep the name of the class as is though